content/subscription/details.md

```diff
@@ -4,19 +4,21 @@ description: Learn about Docker subscription tiers and their features
 keywords: subscription, personal, pro, team, business, features
 ---
 
+A Docker subscription allows the use of Docker components including the Docker CLI, Docker Compose, Docker Engine, Docker Desktop, Docker Hub, Kubernetes, Docker Build and Docker BuildKit, Docker Official Images, and more.
```

Comment:
**Note for reviewer:** Looks like we used `Docker components` in the docs to avoid feature vs product, so I kept that language here and in the overview page but open to feedback on alternative terms.